postgres-appliance/scripts/patroni_wait.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-ROLE=master
+ROLE=primary
```

Comment:
can you plz also change the comment below?
`Waits for ROLE (master or replica).`